packages/error-reporting/src/interfaces/manual.js

```diff
@@ -74,6 +76,22 @@ function handlerSetup(client, config) {
     }
 
     if (err instanceof ErrorMessage) {
+      // The API expects the error to contain a stack trace.  Thus we
+      // append the stack trace of the point where the error was constructed.
+      // See the `message-builder.js` file for more details.
+      if (err._autoGeneratedStackTrace) {
```

Comment:
Is it documented publicly that the ErrorMessage constructor will handle adding stack traces and that user messages should not include them?